scripts/cluster/add_token.py

```diff
@@ -36,6 +38,63 @@ def add_token_with_expiry(token, file, ttl):
             fp.write(token_without_expiry.format(token))
 
 
+def run_util(*args, debug=False):
+    env = os.environ.copy()
+    prog = ["bash", utils_sh_file]
+    prog.extend(args)
+
+    if debug:
+        print("\033[;1;32m+ %s\033[;0;0m" % " ".join(prog))
+
+    result = subprocess.run(
+        prog,
+        stdin=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        env=env,
+    )
+
+    try:
+        result.check_returncode()
+    except subprocess.CalledProcessError as err:
+        raise
```

Comment:
Seems odd to catch this exception only to re-raise it without any action or modification. I assume this is to make it obvious that this section is potentially raising an exception? Maybe it's worth including a comment to that effect to make it clear that it's intentional and not that explicit handling was missed or dropped.

Reply:
There used to be a print statement there indicating there was an issue calling into `utils` and a `sys.exit`. I'll return it to that spec if that is preferred.